pkg/engine/params.go

```diff
@@ -64,6 +64,7 @@ func getParameters(cs *api.ContainerService, generatorCode string, aksEngineVers
 			}
 			if properties.OrchestratorProfile.IsKubernetes() {
 				addValue(parametersMap, "vnetCidr", properties.MasterProfile.VnetCidr)
+				addValue(parametersMap, "masterSubnet", properties.MasterProfile.VnetCidr)
```

Comment:
Why are we not adding this in `params_k8s.go` ?

Reply:
Note to self: Check the pre-conditions for populating `VnetCidr`

Reply:
```go
if properties.MasterProfile.IsCustomVNET() {
	addValue(parametersMap, "masterVnetSubnetID", properties.MasterProfile.VnetSubnetID)
	if properties.MasterProfile.IsVirtualMachineScaleSets() {
		addValue(parametersMap, "agentVnetSubnetID", properties.MasterProfile.AgentVnetSubnetID)
	}
	if properties.OrchestratorProfile.IsKubernetes() {
		addValue(parametersMap, "vnetCidr", properties.MasterProfile.VnetCidr)
		addValue(parametersMap, "masterSubnet", properties.MasterProfile.VnetCidr)
	}
} else {
	addValue(parametersMap, "masterSubnet", properties.MasterProfile.Subnet)
	addValue(parametersMap, "agentSubnet", properties.MasterProfile.AgentSubnet)
}
```
This is the whole part of related code, masterSubnet is added in else condition, I followed this rule to add my code here. Do I need to move it to params_k8s.go?

Reply:
The **vnetCidr** is not used in any condition now.

Reply:
Then I believe we should change that to use VnetCidr, not duplicate values across different parameters.

Reply:
There are a few parameters related to custom VNET, they are  `vnetCidr`/`kubeClusterCidr`/`masterSubnet`/`agentSubnet`. `vnetCidr` is not used, `agentSubnet` is a null value. `kubeClusterCidr` and `masterSubnet` are added to azure cni's exception list only.

I guess there were some thoughts for custom VNET, but not fully implemented yet on Windows. From my side this PR is more as a workaround with minimal change to make things work. 

@PatrickLang  
Any ideas?